src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.builds

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
```

Comment:
I was thinking we just treat it like other shims and put it in the redist.

Alternatively we keep it split, but use regular references instead of runtime references.  That would mean that any fat app that uses it would carry all copies.

I don't think leaving it RID split and requiring a RID is a good option since the mode of failure for a fat app would be runtime failures.

Alternatively SQL could get rid of the sni.dll on Windows and use the managed implementation.  Thought that was in the roadmap but not sure how far off /cc @saurabh500.